include/galsim/hsm/PSFCorr.h

```diff
@@ -122,6 +122,10 @@ namespace hsm {
      *                           KSB method of PSF correction.  Warning: deviating from default
      *                           value of 2 results in code running more slowly, and results have
      *                           not been significantly tested.
+     * @param epsilon            Accuracy (in x0, y0, and sigma as a fraction of sigma) when
```

Comment:
maybe use "x0, y0, and sigma, each as a fraction of sigma"  (if that's accurate).  It wasn't completely clear to me whether the x0 and the y0 were included in the fraction of sigma or not.

Other than that, looks good to me.